addons/main/stringtable.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="SSS">
+	<Package name="Main">
+		<Key ID="STR_SSS_Main_SimplexSupportService">
```

Comment:
This key needs the "s" at the end for it to be more accurate of the title.
`"STR_SSS_Main_SimplexSupportService"` --> `"STR_SSS_Main_SimplexSupportServices"`